bpfman/src/command.rs

```diff
@@ -37,6 +37,7 @@ use crate::{
 };
 
 /// These constants define the key of SLED DB
+pub(crate) const PROGRAM_PREFIX: &str = "program_";
```

Comment:
Since you're changing the schema I'm guessing we'll need to delete the entire DB and re-create it - unless there are DB migrations for sled. We'll need to be mindful of this after we've done a release and perhaps we'll need to codify this somehow.

Reply:
Hrm yeah I'm thinking for right now we're alright... but once we release we'll need to have to have cleanup logic if the schema changes at all 🤔